website/docs/terminology.md

```diff
@@ -84,7 +84,7 @@ The config for the application. It is a dictionary of [Config Nodes](#config-nod
 ## Misc
 ### Config Search Path
 The [Config Search Path](advanced/search_path.md) is a list of paths that are searched in order to find configs. It is similar to
-the Python PYTHONPATH.
+the Python [PYTHONPATH](https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPATH).
```

Comment:
not that similar, you can't override it through environment. dunno if I want that link here.
It is conceptually similar.

Reply:
Yeah I think a better analogy is sys.path

Reply:
sys.path is am implementation detail of how this is done in Python.
this is all modeled after unix PATH.
I think in general it's okay to say it's similar to PYTHONPATH with providing more details.

Reply:
So any change needed ?

Reply:
we can keep this change.